packages/react-native/Libraries/Text/Text/RCTTextShadowView.mm

```diff
@@ -113,11 +132,25 @@ - (void)uiManagerWillPerformMounting
           [descendantViews addObject:descendantView];
         }];
 
+
+    NSMutableArray<RCTVirtualTextView *> *virtualSubviews = [NSMutableArray arrayWithCapacity:virtualSubviewTags.count]; // [macOS]
```

Comment:
this section seemingly should follow with my other comment about #ifdef`ing the code -- if we don't have a build issue, let's not #def limit the change to targeting OSX.  If we do have a build issue, #def all of it (and possibly extract instead of interleaving in an existing function)

Reply:
We ignore `_virtualSubviews` on iOS anyway, so it should be safe to get rid of these ifdef blocks.